tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/NativeMobile/NativeMobileAdditionalInformationControllerTests.cs

```diff
@@ -0,0 +1,120 @@
+using System.Net;
```

Comment:
Not strictly related to this file, but there are extra tests that should be added to, in SolutionGetByIdTests.cs and GetClientApplicationBySolutionIdTests.cs, in Solutions.Application.UnitTests